src/CookingSystem.cpp

```diff
@@ -1248,6 +1248,7 @@ void CookingSystem::CookCommand(CookingCommand& ioCommand, CookingThread& ioThre
 
 	// Store the log output.
 	log_entry.mOutput = output_str.AsStringView();
+	gPreprocessStringViewFormatting(log_entry.mOutput, log_entry.mOutputFormatSpans);
```

Comment:
Maybe a more meaningful name would be something like gParseANSISequences? Or ANSIColors?

If we support more things later we can rename it to something more generic, until then I think it's better to have an explicit name